src/Resources/skeleton/dto/Data.tpl.php

```diff
@@ -0,0 +1,70 @@
+<?= "<?php" . PHP_EOL ?>
+<?php use Symfony\Bundle\MakerBundle\Str;
+
+?>
+
+namespace <?= $namespace ?>;
+
+<?php if (isset($bounded_full_class_name)): ?>
+use <?= $bounded_full_class_name ?>;
+<?php endif ?>
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * Data transfer object for <?= $bounded_class_name ?>.
+ */
+class <?= $class_name ?>
+
+{
+<?php if ($addHelpers): ?>
+    /**
+     * Create DTO, optionally extracting data from a model.
+     */
+    public function __construct(?<?= $bounded_class_name ?> $<?= lcfirst($bounded_class_name) ?> = null)
+    {
+        if ($<?= lcfirst($bounded_class_name) ?> instanceof <?= $bounded_class_name ?>) {
+            $this->extract($<?= lcfirst($bounded_class_name) ?>);
+        }
+    }
+
+    /**
+     * Fill entity with data from the DTO.
+     */
+    public function fill(<?= $bounded_class_name ?> $<?= lcfirst($bounded_class_name) ?>): <?= $bounded_class_name ?><?= PHP_EOL ?>
+    {
+<?php if (false === $omitGettersSetters): ?>
```

Comment:
Could we reverse this - `if ($omitGettersSetters)`? Having the "not" + "omit" is a double-negative - it's a bit confusing :).